NuGet.config

```diff
@@ -18,5 +18,6 @@
   <packageSources>
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
     <add key="applicationinsights" value="https://www.myget.org/F/applicationinsights/api/v3/index.json" />
+    <add key="dotnet" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" protocolVersion="3" />
```

Comment:
will it be available on nugget before the 2.4 release? Would you please create an issue to update the reference and remove this source